plaso/engine/artifact_filters.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+"""Artifact filters processing."""
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from artifacts import definitions as artifact_types
+
+from dfvfs.helpers import file_system_searcher
+from dfwinreg import registry_searcher
+
+from plaso.engine import path_helper
+
+
+ARTIFACT_FILTERS = 'ARTIFACT_FILTERS'
+COMPATIBLE_DFWINREG_KEYS = ['HKEY_LOCAL_MACHINE',
```

Comment:
break after `[`

also should this be a frozenset?

Reply:
Merged in #1883